docs/xml.md

```diff
@@ -561,6 +562,65 @@ A `Ref` value pointing to an `Item` with a `referent` of `RBX466F72207262782D646
 <Ref name="Example">RBX466F72207262782D646F6D21203A2D29</Ref>
 ```
 
+### SecurityCapabilities
+
+The `SecurityCapabilities` data type represents a set of permissions or capabilities, encoded as a bit field, that dictate whether a script can perform specific actions within the Roblox environment. Each bit in the field corresponds to a particular capability.
+
+Roblox uses this type to manage script capabilities for certain instances like `Folder`, `Model`, and `Script`. Although the property is present on all instances, its functionality is only exposed for specific classes. The encoding of `SecurityCapabilities` as a bit field aligns with the permissions previously managed by identity permissions in the lowest 8 bits.
+
+A `SecurityCapabilities` value with no capabilities set appears as follows:
+
+```xml
+<SecurityCapabilities name="Example">0</SecurityCapabilities>
+```
+
+A `SecurityCapabilities` value with `RunClientScript` (2 ^ 8 = 256) & `RunServerScript` (2 ^ 9 = 512) capabilities set appears as follows:
+
+```xml
+<SecurityCapabilities name="Example">768</SecurityCapabilities>
+```
+
+The following table maps each bit in the field to a corresponding capability:
```

Comment:
I am assuming that bit 0 corresponds to the least-significant bit, but we should specify.